test/caesar_test.rb

```diff
@@ -1,2 +1,59 @@
-# YOUR TESTS GOES HERE
-# I PITY THE FOOL WHO DOESNT WRITE TESTS
+
+require 'pry'
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/caesar'
+
+class CaesarTest < Minitest::Test
+
+  def test_if_it_exists
+    caesar = Caesar.new
+    assert_instance_of Caesar, caesar
+    result = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", " "]
+    assert_equal result, caesar.dictionary
+
+  end
+
+  def test_rotate_method
+    caesar = Caesar.new
+    assert_equal 53, caesar.rotate.count
+
+  end
+
+  def test_zipped_array
+    caesar = Caesar.new
+    assert_equal 53, caesar.zipped_array.count
+  end
+
+  def test_for_turn_into_hash
+    caesar = Caesar.new
+    assert_equal 53, caesar.turn_into_hash.count
+
+  end
+
+  def test_if_invert_works
+    caesar = Caesar.new
+    assert_equal 53, caesar.invert_array.count
+  end
+
+  def test_eng_to_cipher
+    caesar = Caesar.new
+    result = "qebXnrf hXZoltkXcluXgrjmpXlsboXqebXiYwvXald"
```

Comment:
This is not what the tests asks for

Reply:
yah but when i asked you if i can put  space into the dictionary u said , thats ok !! now i know i need to ignore space